projects/llm-d/visualizations/llmd_inference/plotting/throughput_comparisons.py

```
@@ -450,6 +450,7 @@ def filter_flavors(setting_lists, flavor_filter):
 
                 def include_ir_flavor(v):
                     if v.startswith('pd-'): return False
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)